CuratedContent/bssw-tutorials.md

```diff
@@ -0,0 +1,26 @@
+### Better Scientific Software (BSSw) Tutorial
+<!-- deck text start --> 
+
+Keeping skills up to date and focused helps keep work at its best. The BSSw Tutorial page presents a vertical sampler of upcoming and past tutorials on improving scientific software.
+
+#### Contributed by [Deborah Stevens](https://github.com/haikudeb)
+#### Publication date: May 08, 2023
+
+Resource information | Details
+:--- | :--- 
+Article title | [The Better Scientific Software (BSSw) Tutorial](https://bssw-tutorial.github.io)
+Authors | [Deborah Stevens](https://github.com/haikudeb)
+Focus | Software Engineering Tutorials
+Website | https://bssw-tutorial.github.io 
+
+
+The website https://bssw-tutorial.github.io contains the information and learning materials needed for best practices in high performance computing. 
+
+Whether a refresher or new insight on high performance computing is needed, this complete collection of all BSSw tutorial presentations (2016-present) exists for your perusal. [Full recordings](https://www.youtube.com/playlist?list=PLuWzStas9iWExOVGGLCUl-N38mxQXSl87) are available as are [presentation slides through FigShare] (https://figshare.com/articles/presentation/Better_Scientific_Software_tutorial_Improving_Scientific_Software_2022_/19416767). Also access to the tutorial-related Hands On Code Repository is a quick click to GitHub : https://github.com/bssw-tutorial/hello-numerical-world-2021-11-15-sc 
+
+The Better Software Scientific Tutorial page is an excellent reference in many ways. Reviewing the best practices for high performance computing via the 
+
+<!---
+Publish: yes
+Topics: Better Skills Better Planning Better Collaboration--Conferences and Workshops
```

Comment:
The way the `Topics:` metadata works is you go look at sub-topics in various of the files [here](https://github.com/betterscientificsoftware/bssw.io/tree/master/Site/Topics) and then copy, verbatim, sub-topic headings that are relevant to the content. In this case, the tutorial covers a lot of ground and so a lot of topics apply...

```suggestion
Topics: Software Engineering, Online Learning, In-Person Learning, Software Process Improvement, Development Tools, Refactoring, Licensing, Testing
```